modules/tinymce/src/core/main/ts/api/dom/RangeUtils.ts

```diff
@@ -60,9 +64,31 @@ const RangeUtils = (dom: DOMUtils): RangeUtils => {
     );
   };
 
+  /**
+   * Expands the range of the selection to contain the entire word when the selection is collapsed within the word
```

Comment:
@brian-forte here's some new API docs you may want to review as well.

Reply:
This description would be correct for `selection.expand()`, but it's not correct for `RangeUtils.expand`...

- [ ] Should this move to selection docs? or be reworded?

Reply:
We really should have API docs on both new APIs to be honest.

Reply:
RangeUtils.expand should talk about ranges and selection.expand should talk about selections so similar to the changelog items.